packages/example-app-base/src/components/ExampleCard/ExampleCard.tsx

```diff
@@ -225,12 +228,16 @@ export class ExampleCardBase extends React.Component<IExampleCardProps, IExample
         this.forceUpdate();
       });
     }
-    if (this.props.onToggleEditor) {
+    if (this.props.onToggleEditor !== undefined && this.props.onToggleEditor) {
```

Comment:
This check should be `this.props.isCodeVisible !== undefined` (and probably move `if (this.props.onToggleCodeEditor)` inside there.

Reply:
Ideally we should also have a warning in the constructor if they provide isCodeVisible!==undefined and no onToggleCode